### DISCUSSION TITLE
Automation for force charging

### DISCUSSION THREAD

**Discussion by jaishkp** [wills106/homeassistant-solax-modbus#392] [General]
I'm wanting to make an automation to change the mode to force charge at 00:31 and back to self-use at 04:29 but I can't find changing to these modes in the action options, any ideas? I think i'm missing something obvious!
Want to figure out how to do this first to then be able to make it more advanced to only activate this automation if the solar forecast for the next day is less than a certain amount and battery is less than a certian amount at 00:31

**Comment by ceemjay** [comment]
I think I might be asking the same question!  I am using a Solis S5-EH1P5K-L with a DSL-L logger - how do I set the 3 charge start and stop times?

I am assuming it is done here:

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/35836468/7230b523-8b37-4811-bd41-54ce092a81a9)

Clicking on the text "Update Charge.." takes me to history and pressing Press does nothing, seemingly. What am I missing?

Many thanks

**Reply by wills106** [reply]
@ceemjay Have a look at the reply I left you here https://github.com/wills106/homeassistant-solax-modbus/discussions/188#discussioncomment-5890852
@jaishkp What Inverter do you own?
